docs/misc/message_system.md

```diff
@@ -165,9 +167,12 @@ Structure of config, types and optionality of specific keys can be found in the
                         {
                             // Possible values: "1" or "T"
                             "model": "1",
+                            // one of firmware and bootloader has to be *
                             "firmware": "1.9.4",
+                            "bootloader": "*",
                             // Possible values: "*", "bitcoin-only", and "regular"
                             "variant": "bitcoin-only",
+                            "revision": "*",
```

Comment:
Should we call this fw_revision so it is clearly distinguishable from bl_revision?

Reply:
625b8e6b09db6932fa5064db296f1431025e3b65